tests/suites/test_suite_cipher.function

```diff
@@ -163,8 +175,20 @@ void enc_dec_buf( int cipher_id, char *cipher_string, int key_len,
     TEST_ASSERT( 0 == mbedtls_cipher_setup( &ctx_dec, cipher_info ) );
     TEST_ASSERT( 0 == mbedtls_cipher_setup( &ctx_enc, cipher_info ) );
 
-    TEST_ASSERT( 0 == mbedtls_cipher_setkey( &ctx_dec, key, key_len, MBEDTLS_DECRYPT ) );
-    TEST_ASSERT( 0 == mbedtls_cipher_setkey( &ctx_enc, key, key_len, MBEDTLS_ENCRYPT ) );
+    ret = mbedtls_cipher_setkey( &ctx_dec, key, key_len, MBEDTLS_DECRYPT );
+    if( ( ret == MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_CAMELLIA_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_GCM_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_DES_FEATURE_UNAVAILABLE ) )
+        goto exit;
+    TEST_ASSERT( 0 == ret );
+    ret = mbedtls_cipher_setkey( &ctx_enc, key, key_len, MBEDTLS_ENCRYPT );
+    if( ( ret == MBEDTLS_ERR_AES_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_CAMELLIA_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_GCM_FEATURE_UNAVAILABLE ) ||
+        ( ret == MBEDTLS_ERR_DES_FEATURE_UNAVAILABLE ) )
+        goto exit;
```

Comment:
No action required, but just curious:
Is there a way to say a test was skipped yet?

Reply:
not with this infrastructure, unfortunately.
The skip is done in the `main_test.function`, and until we have a common `FEATURE_UNAVAILABLE` error, we can't determine this.